packages/fxa-content-server/app/scripts/lib/glean/index.ts

```diff
@@ -45,7 +52,7 @@ const hashUid = async (uid) => {
   return hex;
 };
 
-const populateProperties = async () => {
+const populateMetrics = async () => {
```

Comment:
I  renamed this so since we are going to refer to the additional event context as event properties.  "metrics" is also more aligned with how Glean refers to these values anyway.